src/content/docs/en/setup/getting.mdx

```diff
@@ -32,38 +27,29 @@ Ultramarine Linux is distributed in various formats available for download from
-We see immutability as the future of Linux systems. Ideally, this will become the standard edition in the future.
+With Ultramarine 41 we are now providing a developer preview of our bootc variant. Ultramarine bootc is based on Fedora bootc, but we intend to switch to our own base container (based on Fedora packages) in the future.
+Bootc. We see this as the future of Linux systems. Ideally, this will become our standard edition in the future.
 
-These builds are currently in beta. If you are planning to test them, please join our [Discord](https://discord.gg/5fdPuxTg5Q) or [Matrix](https://matrix.to/#/#hub:fyralabs.com) community first and let us know. We'll be happy to receive feedback and assist you.
+These builds are not ready for daily use by the average user. If you are planning to test them, please join our [Discord](https://fyralabs.com/discord) or [Matrix](https://fyralabs.com/matrix) community and keep us posted on any bugs you find, from a papercut to a broken system.
 
-[Try Ultramarine Atomic →](https://nightly.link/Ultramarine-Linux/ostree-config/workflows/ci-isos.yaml/main?preview)
+[Try Ultramarine bootc →](todo)
 
-(The "Home" and "Base" images are copies of [tauOS](https://tauos.co) - These **ARE NOT READY FOR DAILY USE!!**)
+We previously offered OSTree based builds of Ultramarine Atomic, these are no longer maintained.
 
 ### Raw Images and Raspberry Pi
 
 Ultramarine is also availible as a raw (or preinstalled) image. These images come with a desktop preinstalled and are intended to be flashed directly to the drive your device will boot from (like an SSD or an SD card.)
 
-These can be found in the [Raw Image](https://images.fyralabs.com/images/ultramarine/40/) sections of the download page.
+These can be found in the [Raw Image](https://images.fyralabs.com/images/ultramarine/41/) sections of the download page.
 
 <Alert type="info">
   If you're using a Raspberry Pi 4, you can skip these steps and use Raspberry
   Pi Imager! Either insert an empty microSD and press shfit at startup, or use
   it on another device.
 </Alert>
 
-{/* ### Chromebook Edition */}
-
-{/* Ultramarine is available for most x86 ChromeOS devices. See the [Chromebook Edition section](/en/chromebook/chromebook) for more information. */}
-
-{/* ### Kanopy */}
-
-{/* **Ultramarine Kanopy** is an upcoming variant of Ultramarine Linux, which is made for servers and cloud computing. It is currently in development and does not have a release date yet, but we plan to target Qx '2x */}
-{/* It will be based on Ultramarine OSTree and is designed as a simple and easy-to-use server operating system, with a focus on containerized applications with a simple, easy-to-use web interface as a competitor to TrueNAS SCALE, OpenMediaVault, and other similar operating systems. */}
-

```

Comment:
is this included somewhere else?

Reply:
no, we didn't publish any of these, think we left them for the future

Reply:
so… no more kanopy? discontinued?

Reply:
cappy question